devscripts/changelog_override.json

```diff
@@ -0,0 +1,74 @@
+[
```

Comment:
Note: Remember to delete before merge

Reply:
This can be deleted whenever (now?) and can be tested using `actions-test`, fork or locally if you want to test results. Should be populated though if things need to be corrected before merge.